Source/AdMobAdapterBannerAd.swift

```diff
@@ -28,8 +28,19 @@ class AdMobAdapterBannerAd: AdMobAdapterAd, PartnerBannerAd {
             completion(.failure(error))
             return
         }
-        
-        let bannerView = GADBannerView(adSize: gadAdSize(from: request.bannerSize))
+
+        // Fail if we cannot fit a fixed size banner in the requested size.
+        guard
+            let requestedSize = request.bannerSize,
+            let fittingSize = BannerSize.largestStandardFixedSizeThatFits(in: requestedSize),
```

Comment:
I think this might be wrong.
We don't want to "round off" an adaptive banner to the nearest standard size.